1.9/usage/jobs/examples.md

```diff
@@ -5,56 +5,92 @@ feature_maturity: preview
 menu_order: 20
 ---
 
-# A Simple Job
-
-```json
-{
-  "id": "my-job",
-  "description": "A job that sleeps",
-  "run": {
-    "cmd": "sleep 1000",
-    "cpus": 0.01,
-    "mem": 32,
-    "disk": 0
-  }
-}
-```
-
-# A Job with a Schedule
-
-```
-{
-  "id": "my-scheduled-job",
-  "description": "A job that sleeps on a schedule",
-  "run": {
-    "cmd": "sleep 20000",
-    "cpus": 0.01,
-    "mem": 32,
-    "disk": 0
-  },
-  "schedules": [
+<a name="create-job"></a>
+# Create a Simple Job
+
+This JSON creates a simple job with no schedule.
+
+1. Create a JSON file with the following contents. 
+    ```json
+    {
+      "id": "my-job",
+      "description": "A job that sleeps",
+      "run": {
+        "cmd": "sleep 1000",
+        "cpus": 0.01,
+        "mem": 32,
+        "disk": 0
+      }
+    }
+    ```
+
+1. Add the job from the DC/OS CLI.
+    ```bash
+    dcos job add <my-job>.json
+    ```
+    
+    Alternatively, add the job using the API.
+    ```bash
+    curl -X POST -H "Content-Type: application/json" -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/service/metronome/v1/jobs -d@/Users/<your-username>/<myjob>.json
+    ```
+
+# Create a Job with a Schedule via the DC/OS CLI
+**Note:** This example JSON only works when you add the job from the DC/OS CLI or the UI. Use the [example below](#schedule-with-api) to create a scheduled job via the API.
```

Comment:
These are instructions for creating a job with a schedule from the CLI. But the note accurately states that jobs with schedules can be created with the CLI and the UI, just not the API. I could take out the mention of the UI, but then I'd be implying that you can't create scheduled jobs that way.